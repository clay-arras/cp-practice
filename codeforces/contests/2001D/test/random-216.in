1
18
15 15 4 17 11 5 6 2 14 25 15 6 21 6 12 12 8 3
