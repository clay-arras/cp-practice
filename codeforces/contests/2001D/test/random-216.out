13
15 4 17 11 5 2 14 25 21 6 12 8 3
